core/src/main/java/com/kodebeagle/javaparser/MethodInvocationResolver.java

```diff
@@ -67,6 +72,19 @@
         return declaredMethods;
     }
 
+    public Set<TypeJavadoc> getTypeJavadocs() {
+
+        Set<TypeJavadoc> typeJavadocsSet = new HashSet<>();
+
+        for (String key : this.typeJavadocs.keySet()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Iterate over the "entrySet" instead of the "keySet". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2864)